macantine/tasks.py

```diff
@@ -148,6 +151,22 @@ def no_diagnostic_first_reminder():
                 )
 
 
+def _covered_by_central_kitchen(canteen):
+    if canteen.production_type == Canteen.ProductionType.ON_SITE_CENTRAL and canteen.central_producer_siret:
+        try:
+            central_kitchen = Canteen.objects.get(siret=canteen.central_producer_siret)
+            covered_by_central_kitchen = central_kitchen.diagnostic_set.filter(
+                central_kitchen_diagnostic_mode=Diagnostic.CentralKitchenDiagnosticMode.ALL,
+            ).exists()
+            if covered_by_central_kitchen:
+                return True
```

Comment:
Nitpicking : could be `return covered_by_central_kitchen` but not that important to change it tbh

Reply:
Done here : https://github.com/betagouv/ma-cantine/pull/2149/commits/1d7b32f6a1cd82c0ca50e9df50eff9b42a6c3ea2